# Point cloud holding the sensed object 3D data
sensor_msgs/PointCloud2 cloud

# Object bounding box's min and max x,y,z
geometry_msgs/PointStamped boundingBoxMin
geometry_msgs/PointStamped boundingBoxMax